### PR TITLE
Added chnges in readme file

### PR DESCRIPTION
**Pull request by siddharth1729**
Please review this and provide guidance on any additional steps I should take.

I'm eager to contribute to this project as a backend developer who's deeply interested in databases and low-level technologies. I'm actively learning and find this repository particularly intriguing.

It would be beneficial to have a Discord group where contributors can ask questions, gain a better understanding of the project's concepts, align their efforts with project ideas, and engage in casual conversations.

Looking forward to your insights and suggestions.

https://thelegion.tech is my profilio to know more about me.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,9 +9,23 @@ It provides a new index type for vector columns called `hnsw` which speeds up `O
...
+### Requirements
 
-LanternDB builds and uses [usearch](https://github.com/unum-cloud/usearch) for its single-header state of the art HNSW implementation.
+Make sure you have the following software installed:
+
+- `gcc` (version >= ?)
+- `g++` (version >= ?)
+- `postgres` (version >= ?)
+- `cmake` (version >= ?)
+- `make` (version >= ?)
```
Comment (Ngalstyan4) [inline README.md:20]: These do not go in the main readme. We will have them in our docs. 
However, if you have old versions of each and could help us on which versions LanternDB does not compile, that would be helpful.

File: README.md [hunk 2]
```diff
@@ -9,9 +9,23 @@ It provides a new index type for vector columns called `hnsw` which speeds up `O
...
+
+#### Installing pgvector
+
+Before proceeding, you need to install [pgvector](https://github.com/pgvector/pgvector), which is a dependency for LanternDB.
```
Comment (Ngalstyan4) [inline README.md:24]: This will soon be unnecessary. no need to add it to the readme.

**Review by Ngalstyan4 — COMMENTED** [review]
Thanks for expressing interest, @siddharth1729!
We will have a Discord soon. Will e-mail you in the meantime.

File: README.md (hunk 1 above)
Comment (siddharth1729) [reply README.md:20]: ohk will remove this for time being

File: README.md (hunk 2 above)
Comment (siddharth1729) [reply README.md]: Done pleas have a look @Ngalstyan4

**Comment by Ngalstyan4** [comment]
Thanks for your contribution, @siddharth1729!

**Pull request merged**